src/legacy/core_plugins/embeddable_api/public/panel/panel_header/panel_actions/add_panel/add_panel_flyout.tsx

```diff
@@ -38,6 +38,7 @@ import {
   EuiText,
 } from '@elastic/eui';
 
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
 import { SavedObjectAttributes } from 'src/core/server/saved_objects';
```

Comment:
I think `src/**` imports that used Webpack alias, like

```
import 'src/core/public/mocks';
```

where not covered previously, now this PR fixes it.

![image](https://user-images.githubusercontent.com/9773803/61236672-de3c1f00-a738-11e9-8474-95d60b1dbef3.png)

Added a test for it, too.